pages/api/specificApy.js

```diff
@@ -66,7 +74,7 @@ function getProvider(chain = 1) {
 		} else {
 			return new ethers.providers.InfuraProvider('homestead', '9aa3d95b3bc440fa88ea12eaa4456161');
 		}
-	} else if (chain === 'polygon') {
+	} else if (chain === 'polygon' || chain === 137) {
```

Comment:
why 137?

Reply:
137 is the chainID for Polygon. This function should accept number and not string, so I added this to be able to migrate that :)